mitk-plugin/Plugins/org.mitk.nvidia.aiaa/src/NvidiaSmartPolySegTool2DGUI.cpp

```diff
@@ -77,17 +74,24 @@ void NvidiaSmartPolySegTool2DGUI::OnNewToolAssociated(mitk::Tool *tool) {
     }
 
     // convert current segmentation mask to polygon
-    m_NvidiaSmartPolySegTool2D->Mask2Polygon();
+    QApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
+    try {
+      m_NvidiaSmartPolySegTool2D->Mask2Polygon();
+      Refetch();
+    } catch (...) {
+    }
+    QApplication::restoreOverrideCursor();
   }
 }
 
 void NvidiaSmartPolySegTool2DGUI::SetStepper(mitk::Stepper *stepper) {
-  updateConfigs();
   this->m_SliceStepper = stepper;
```

Comment:
what is this SetStepper for?
why we don't need updateConfigs anymore?

Reply:
yes.. this callback only sets the object... no ui action happens here..